website/styles/index.css

```
@@ -345,5 +345,92 @@ body:not(.has-sidebar) .page-center {
+    padding: 10px 14px;
+    font-size: 16px;
+  }
+}@media (max-width: 1024px) {
+  .sidebar {
+    display: none;
+  }
+
+  body.has-sidebar main {
+    margin-left: 0 !important;
+  }
+}img {
```

Stylelint (17.3.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)